npm/cypress-schematic/src/schematics/ng-add/index.spec.ts

```diff
@@ -32,15 +32,13 @@ describe('@cypress/schematic: ng-add', () => {
   })
 
   it('should create cypress files', async () => {
-    const files = ['cypress/e2e/spec.cy.ts', 'cypress/plugins/index.ts', 'cypress/support/commands.ts', 'cypress/tsconfig.json', 'cypress.config.ts']
-    const homePath = '/projects/sandbox/'
-
     return schematicRunner.runSchematicAsync('ng-add', {}, appTree).toPromise().then((tree) => {
-      files.forEach((f) => {
-        const pathToFile = resolve(homePath, f)
+      const files = tree.files
 
-        expect(tree.exists(pathToFile), pathToFile).equal(true)
-      })
+      expect(files).to.contain('/projects/sandbox/cypress/e2e/spec.cy.ts')
```

Comment:
Would it be helpful to add assertions for `should not contain` the files that we removed ie: `plugins, cypress.json`